File: app/src/main/java/com/lany/fivechess/game/Point.java
package com.lany.fivechess.game;

/**
 * 坐标类
 */
public class Point {
    private int X;
    private int Y;

    public Point() {

    }

    public Point(int x, int y) {
        X = x;
        Y = y;
    }

    public int getX() {
        return X;
    }

    public void setX(int x) {
        X = x;
    }

    public int getY() {
        return Y;
    }

    public void setY(int y) {
        Y = y;
    }
}

File: app/src/main/java/com/github/lany192/fivechess/game/Point.kt
package com.github.lany192.fivechess.game

/**
 * 坐标类
 */
class Point {
    private var x: Int = 0
    private var y: Int = 0

    constructor() {

    }

    constructor(x: Int, y: Int) {
        this.x = x
        this.y = y
    }

    fun setX(x: Int) {
        this.x = x
    }

    fun getX(): Int {
        return x
    }

    fun setY(y: Int) {
        this.y = y
    }

    fun getY(): Int {
        return y
    }

}